example/src/components/Stats.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  Collapse,
+  CollapseHeader,
+  CollapseBody,
+} from 'accordion-collapse-react-native';
+import React, { useCallback } from 'react';
+import { View, StyleSheet, processColor } from 'react-native';
+import { LineChart } from 'react-native-charts-wrapper';
+
+import { Typo } from './Typo';
+
+type StatsProp = {
+  stats: any[];
+  label: string;
+};
+
+const notPlottableStats = [
+  'kind',
+  'rid',
+  'packetsLost',
+  'packetsReceived',
+  'bytesReceived',
+  'framesReceived',
+  'framesDropped',
+  'bytesSent',
+  'packetsSent',
+  'framesEncoded',
+];
+
+const chartDescriptionConfig = {
+  text: '',
+  textSize: 0,
+};
+
+const yAxisConfig = {
+  left: {
+    enabled: false,
+  },
+};
+
+export const Stats = ({ stats, label }: StatsProp) => {
+  const getValues = useCallback(
+    (label: string, chart: string) => {
+      const timestampsWithLabel = stats.filter((obj) => {
+        return Object.keys(obj).includes(label);
+      });
+
+      const values = timestampsWithLabel.map((obj) => {
+        return { y: obj[label][chart] !== null ? obj[label][chart] : 0 };
+      });
+
+      return values;
+    },
+    [stats]
+  );
+
+  const generateLimitationDurationInitData = useCallback(
+    (name: string, color: string) => {
+      return {
+        label: name,
+        values: [{ y: 0 }],
+        config: {
+          drawValues: false,
+          drawCircles: false,
+          color: processColor(color),
+        },
+      };
+    },
+    []
+  );
+
+  const getLimitationDurationsDataset = useCallback(
```

Comment:
Could you explain what do those numbers mean? For example, I don't know what bandwidth limitation duration = 700 means. What are the units here, seconds?

Reply:
A record of the total time, in seconds, that this stream has spent in each quality limitation state.